udata_gouvfr/templates/nec_mergitur.html

```diff
@@ -0,0 +1,85 @@
+{% extends theme("base.html") %}
+
+{% set meta = {
+    'title': 'Nec Mergitur',
+    'description': "Un Hackathon pour renforcer la sécurité dans Paris",
+    'keywords': [
+        'necmergitur',
+        'paris',
+        'fluctuat',
+        'nec',
+        'mergitur',
+        'hackathon',
+        'évènement',
```

Comment:
https://fr.wikipedia.org/wiki/Rectifications_orthographiques_du_fran%C3%A7ais_en_1990
The old school people we are would choose "événement" but we need to get over it: https://fr.wikipedia.org/wiki/Rectifications_orthographiques_du_fran%C3%A7ais